Python/README.md

```diff
@@ -337,20 +338,21 @@ S. No | Name of Script | Directory Link |  Author | Tools Used |
-162 | Youtube Video Download | [Click Here](/Python/Youtube_Video_Download) | [Anushka-S10](https://github.com/Anushka-S10) | pytube
-163 | YouTube Video-Audio Download | [Click Here](/Python/YouTube_Video-Audio_Download) | [Naman Shah](https://github.com/namanshah01) | os, sys, pafy, requests
-164 | Zipper | [Click Here](/Python/Zipper) | [Puneet Pal Singh](https://github.com/Puneet-Pal-Singh) | os, zipfile, datetime
-165 | Zoom Automation | [Click Here](/Python/Zoom_Automation) | [Rutuja Dhanawade](https://github.com/rutujadhanawade) | pyautogui, time
+150 | View and Combine STL | [Click Here](/Python/View_and_Combine_STL) | [Archan Ghosh](https://github.com/ArchanGhosh) | numpy-stl, numpy, matplotlib, glob
+151 | Vigenere Cipher | [Click Here](/Python/Vigenere_Cipher) | [Mohit Gupta](https://github.com/mohitacecode) | -
+152 | Watermark It | [Click Here](/Python/Watermark_it) | [Aditya Jetely](https://github.com/AdityaJ7) | pdfrw, sys, os
+153 | Website Blocker | [Click Here](/Python/Website_Blocker) | [Ankita Priya](https://github.com/AnkitaBIT) | datetime
+154 | Website Cloner | [Click Here](/Python/Website_Cloner) | [Avinash Kr. Ranjan](https://github.com/avinashkranjan) | logging, cssutils, BeautifulSoup
+155 | Weather Using Open Weather Map | [Click Here](/Python/Weather_using_OpenWeatherMap) | [Namya LG](https://github.com/Namyalg) | requests, json, sys
+156 | WhatsApp Message Automation | [Click Here](/Python/WhatsApp_Message_Automation) | [Namya LG](https://github.com/Namyalg) | selenium, time
+157 | WhatsApp SMS Bomber | [Click Here](/Python/Whatsapp_Sms_Bomber) | [ANSHUMALI SHAW](https://github.com/bagofcodes) | selenium, tkinter, xpath
+158 | Who is Online On WhatsApp | [Click Here](/Python/Who_Is_Online_On_WhatsApp) | [Wable Sanket](https://github.com/Sanketwable) | selenium, time, users
+159 | Word Meaning | [Click Here](/Python/Word_Meaning) | [Mohit Gupta](https://github.com/mohitacecode) | PyDictionary
+160 | Wifi Spped Tester | [Click Here](/Python/Wifi%20Speed%20Tester) | [Vedansh Vijaywargiya](https://github.com/vvHacker007) | win10toast, speedtest
+161 | XML To CSV | [Click Here](/Python/XML_To_CSV) | [Harsh Bardhan Mishra](https://github.com/HarshCasper) | glob, pandas
+162 | Youtube Bot | [Click Here](/Python/Youtube_Bot) | [Sri Manikanta Palakollu](https://github.com/srimani-programmer) | time, argparse, selenium
+163 | Youtube Video Download | [Click Here](/Python/Youtube_Video_Download) | [Anushka-S10](https://github.com/Anushka-S10) | pytube
+164 | YouTube Video-Audio Download | [Click Here](/Python/YouTube_Video-Audio_Download) | [Naman Shah](https://github.com/namanshah01) | os, sys, pafy, requests
+165 | Zipper | [Click Here](/Python/Zipper) | [Puneet Pal Singh](https://github.com/Puneet-Pal-Singh) | os, zipfile, datetime
+166 | Zoom Automation | [Click Here](/Python/Zoom_Automation) | [Rutuja Dhanawade](https://github.com/rutujadhanawade) | pyautogui, time
```

Comment:
You can revert your commits for the Python README. I have delivered a fix right now, which might create some merge conflicts. Its better to give it a skip and come up with a more friendly approach of sorting them out.

Reply:
Changes Reverted